zulipterminal/ui_tools/utils.py

```diff
@@ -35,3 +35,54 @@ def create_msg_box_list(model: Any, messages: Union[None, Iterable[Any]]=None,
     if focus_msg is not None:
         model.index['pointer'][str(model.narrow)] = focus_msg
     return w_list
+
+
+HELP_TEXT = """\
```

Comment:
I wonder if there's a way to make this be populated dynamically -- currently, we have to maintain these keybinding in 3 places - the actual code, this help text and the README.  It might be easier to maintain if we are able to make the keybindings a little more "self documenting".